Generator/AssetCategoryGenerator.php

```diff
@@ -29,11 +29,14 @@ class AssetCategoryGenerator implements GeneratorInterface
      *
      * {@inheritdoc}
      */
-    public function generate(array $config, $outputDir, ProgressHelper $progress, array $options = null)
+    public function generate(array $globalConfig, array $config, ProgressHelper $progress, array $options = [])
     {
         $this->locales = $options['locales'];
 
-        $faker = Faker\Factory::create();
+        $faker = \Faker\Factory::create();
```

Comment:
Shouldn't we provide a factory shared between generators instead of reusing the same code ?